usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -0,0 +1,227 @@
+                # Note: we could of included indexes above (`INCLUDING CONSTRAINTS`) but we need to drop the
+                #       foreign key ones
+                if source_table:
+                    logger.info(f"Copying constraints over from {source_table}")
+                    copy_constraint_sql = make_copy_constraints(
+                        cursor, source_table, temp_destination_table, drop_foreign_keys=True
+                    )
+                    if copy_constraint_sql:
+                        cursor.execute("; ".join(copy_constraint_sql))
+                    logger.info(f"Constraints from {source_table} copied over")
+
+            # Converting unsupported data types on spark to their delta equivalent
+            # We will cast these back to their original intended types afterwards
+            # Only applicable if we're copying down a table that was already in postgres
+            alter_column_sql = []
+            source_columns = []
+            if source_model:
+                source_columns = [(column.name, type(column)) for column in source_model._meta.get_fields()]
+            else:
+                source_columns = list(source_schema.items())
+
+            for column_name, column_type in source_columns:
+                if column_type in SPECIAL_TYPES_MAPPING:
+                    special_columns[column_name] = column_type
+                    delta_type = SPECIAL_TYPES_MAPPING[column_type]["delta"].format(column_name=column_name)
+                    alter_column_sql.append(
+                        f"ALTER TABLE {temp_destination_table}" f" ALTER COLUMN {column_name} TYPE {delta_type}"
+                    )
+            if alter_column_sql:
+                with db.connection.cursor() as cursor:
+                    cursor.execute(";".join(alter_column_sql))
+
+        # Read from Delta
+        df = spark.table(delta_table)
+
+        # Make sure that the column order defined in the Delta table schema matches
+        # that of the Spark dataframe used to pull from the Postgres table. While not
+        # always needed, this should help to prevent any future mismatch between the two.
+        if table_spec.get("column_names"):
+            df = df.select(table_spec.get("column_names"))
+
+        # Write to Postgres
+        logger.info(f"LOAD (START): Loading data from Delta table {delta_table} to {temp_destination_table}")
+        df.write.options(customSchema=custom_schema, truncate=True).jdbc(
+            url=jdbc_url, table=temp_destination_table, mode="overwrite", properties=get_jdbc_connection_properties()
+        )
+        logger.info(f"LOAD (FINISH): Loaded data from Delta table {delta_table} to {temp_destination_table}")
+
+        if source_table or source_schema:
+            # Recast the special case columns to their postgres equivalents
+            alter_column_sql = []
+            for column_name, column_type in special_columns.items():
```

Comment:
Since this may have the potential of a long wait while it executes, it'd be good to put a log statement in here, before the for loop:
e.g. 
f"Altering columns back to their postgres-side types for columns: {special_columns}"